microsoft-edge/devtools-guide-chromium/memory-problems/memory-101.md

```diff
@@ -23,222 +23,175 @@ ms.date: 07/21/2023
 
-The memory graph starts with a root, which may be the `window` object of the browser or the `Global` object of a Node.js module.  You don't control how the root object is garbage-collected.
+The memory graph starts with a GC root, which may be the `window` object of the browser or the `Global` object of a Node.js module. You don't control how the root object is garbage-collected:
 
 ![You can't control how the root object is garbage-collected](./memory-101-images/dontcontrol.png)
 
-Whatever isn't reachable from the root gets garbage-collected.
-
-> [!NOTE]
-> The number that's shown in the [Shallow size](#shallow-size) and [Retained size](#retained-size) columns is the number of bytes.
+Nodes that aren't reachable from the root can get garbage-collected.
 
 
 <!-- ====================================================================== -->
-## Objects retaining tree
-
-The heap is a network of interconnected objects.  In the mathematical world, this structure is called a _graph_ or _memory graph_.  A graph is constructed from _nodes_ that are connected by _edges_.
+## Object sizes and distances
 
-Nodes and edges in a graph are given labels as follows:
+When working with the **Memory** tool, you will likely find yourself looking at the following columns of information:
 
-*  _Nodes_ (or _objects_) are labelled using the name of the _constructor_ function that was used to build them.
+* **Shallow Size**
+* **Retained Size**
+* **Distance**
 
-*  _Edges_ are labelled using the names of _properties_.
+The numbers in the **Shallow size** and **Retained size** columns are the number of bytes.
 
-Learn [how to record a profile using the Heap Profiler](heap-snapshots.md).  In the following figure, some of the notable things in the Heap Snapshot recording in the [Memory](heap-snapshots.md) tool include **Distance**, which is the distance from the garbage-collection root.  If almost all the objects of the same type are at the same distance, and a few are at a bigger distance, that's something worth investigating.
+![Shallow and Retained size](./memory-101-images/shallow-retained.png)
 
-Distance from root:
+<!-- ------------------------------ -->
+#### Shallow size
 
-![Distance from root](./memory-101-images/root.png)
+The _shallow size_ is the size of the memory that is held by the object directly.
 
+The shallow size of an object is usually small because JavaScript objects often only store the descriptions of the object in memory, and not their values. Arrays and strings are exceptions to this and can have a significant shallow size. Most JavaScript objects store their values elsewhere in the renderer memory, and only expose a small wrapper object on the JavaScript heap.
```

Comment:
```suggestion
The shallow size of an object is usually small, because JavaScript objects often only store the descriptions of the object in memory, and not their values. Arrays and strings are exceptions to this, and can have a significant shallow size. Most JavaScript objects store their values elsewhere in the renderer memory, and only expose a small wrapper object on the JavaScript heap.
```
commas to help parse.  comma = when reading aloud, a pause.